tools/debian/control

```diff
@@ -104,9 +104,8 @@ Architecture: all
 Multi-Arch: foreign
 Depends: ${misc:Depends},
          libvirt-daemon-system,
-         libvirt-clients | libvirt-bin,
-         libosinfo-bin
-Recommends: virtinst, libvirt-dbus
+         libvirt-clients | libvirt-bin
+Recommends: virtinst, libvirt-dbus, python3-gi, libosinfo-1.0-dev
```

Comment:
Likewise, no need for the -dev package; gir1.2-libosinfo-1.0 is enough (for the typelib).